pyro/optim/dct_adam.py

```diff
@@ -103,7 +104,7 @@ def step(self, closure=None):
 
         return loss
 
-    def _step_param(self, group, p):
+    def _step_param(self, group: Dict, p) -> None:
```

Comment:
I din't get what p is and the datatype is expected?

Reply:
I believe `p` is the parameter to be differentiated and is of type `torch.Tensor` (it may or may not be the narrower type `torch.nn.Parameter`)